scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/binary-search-tree.py

```diff
@@ -0,0 +1,83 @@
+class TreeNode:
+    def __init__(self, key):
+        self.key = key
```

Comment:
Seems like `number` or `value` might be better name, instead of `key`. Example at the bottom uses only numbers, but technically there shouldn't be issues with using it for any class that supports comparison between objects from that class.

Reply:
I think using `key` is fine. In the challenge step description we can mention that, although only numerical keys will be used in the project.